### ISSUE TITLE
Incubation status

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/responsive-image-client-hints#25]
Hi @eeeps! Just checking for status on this incubation as it looks kinda dormant (not much activity for a few years). 

Is this something you want to continue to pursue or should we archive this?

**Comment by yoavweiss** [comment]
This is not something we should archive, but instead, we should clearly [mark](https://github.com/WICG/admin/issues/138) it as "shipped in a single implementation" (or something along those lines).

**Comment by eeeps** [comment]
Apologies for the late response. Basically, what @yoavweiss said. This spec may be Dormant, but it isn't dead!

There are some important open issues (#19, #4, and #24 are the highest on my list) but without feedback from implementers (both UAs, and other folks running servers/CDNs), I have abstained from bikeshedding my own answers, disconnected from any kind of consensus.

IMO this spec is valuable to mark as not dead because it:

1. documents what Blink does
2. explains/frame the importance of the features
3. is a place for interested parties to raise questions and provide feedback. 

That all feels valuable, and if/when another browser implements Client Hints Infrastructure (which has been very active), I hope to advocate for this spec more actively.

Open to any thoughts and I'll be keeping an eye on https://github.com/WICG/admin/issues/138.

**Comment by jonarnes** [comment]
Yes, agreed. Simply because responsive images is not fully solved yet, this proposal has value in providing a potential solution.
